packages/shared/lib/src/sdk/tx.rs

```diff
@@ -445,6 +447,11 @@ fn tx_msg_into_args(tx_msg: &[u8]) -> Result<args::Tx, JsError> {
         _ => None,
     };
 
+    let memo = match memo {
```

Comment:
You can also map Option:
```rs
    let memo = memo.map(|v| v.as_bytes().to_vec());
```

Reply:
Nice, thanks!

Reply:
@mateuszjasiuk RE: You're question about validating Memo - this can be any string, and I did some testing to ensure that it was deserializing into the correct Tx args, however, we will need to validate Tx size when using this field with Ledger, so a large `memo` could cause that to fail. Looking at that in a new PR!